### PR TITLE
actualize archive for fire-squid

### PR DESCRIPTION
**Pull request by dzhelezov**

### CODE REVIEW

File: docs/archives/archives-advanced-setup.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+For Substrate-based chains, an Archive consists of a Postgres compatible database for storage,
+the ingest service is [substrate-ingest](https://github.com/subsquid/squid/tree/master/substrate-ingest), and the data is exposed by [substrate-gateway](https://github.com/subsquid/archive-gateway)
+and optionally by [substrate-explorer](https://github.com/subsquid/squid/tree/master/substrate-explorer) for human-friendly exploration queries. 
+
+## Substrate Ingest
+
+`substrate-ingest` fetches blocks from a substrate chain and dumps them into a Postgres-compatible database or as JSON lines. It has the following arguments 
+
+- `-e`, `--endpoint` A wss RPC endpoint. One can provide as many endpoints as necessary, increasing the ingestion throughput.
+- `-c`, `--capacity` (optional, `default: 5`. should follow the endpoint) capacity. The max number of pending requests for the endpoint/
```
Comment (eldargab) [inline docs/archives/archives-advanced-setup.md:12]: `--endpoint-capacity`

File: docs/archives/archives-advanced-setup.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- `--types-bundle`, A path to a local JSON file with substrate type definitions (applicable only to old pre-v14 metadata substrate chains). Note that the types bundle format is [slight different](https://github.com/subsquid/squid/tree/master/substrate-metadata/src/old/definitions) than that of `polkadot.js`
```
Comment (eldargab) [inline docs/archives/archives-advanced-setup.md:13]: It's worth noting, that for many parachains we have built in type bundles. For such chains this option is advised to be omitted.

File: docs/archives/new-in-fire-squid.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# What's new in FireSquid Archives
+
+The FireSquid release of Squid Archives brought about significant performance improvements:
+
+- FireSquid archives can concurrently source blocks from multiple endpoints. The synchronization can be as fast as 2000 blocks per second and is bottlenecked only by the database writes.
```
Comment (eldargab) [inline docs/archives/new-in-fire-squid.md:5]: `2000 bps` - Was that really achieved?

Comment (dzhelezov) [reply docs/archives/new-in-fire-squid.md]: Ok, dialed back to 500 :)

**Pull request merged**